session/session_test.go

```diff
@@ -3928,7 +3929,7 @@ func (s *testBackupRestoreSuite) TestBackupAndRestore(c *C) {
 		tmpDir := path.Join(os.TempDir(), "bk1")
 		os.RemoveAll(tmpDir)
 		// backup database to tmp dir
-		tk.MustQuery("backup database * to 'local://" + tmpDir + "'")
+		tk.MustQuery("backup database br to 'local://" + tmpDir + "'")
```

Comment:
Why has this test changed?

Reply:
We found the latest BR will make this test unstable,  We tried to reproduce by update go mod in #26941, and we re-produced it. https://ci.pingcap.net/blue/organizations/jenkins/tidb_ghpr_check_2/detail/tidb_ghpr_check_2/23358/pipeline
I'm working on fixing it, this change can avoid the unstable test.

Reply:
Got it, thanks.

Reply:
We found the root cause, because in https://github.com/pingcap/br/pull/1402 br introduced 7s backoff of scatter region. 
and in session_test, there is only one tikv and one pd. so every region will backoff 7s due to `region not replicated during scatter`.

We have  back up  50+ region in test, it will cause 350+ second to finish. 

For short term,  In this PR change the `*` to `br ` will reduce the region count from 50 to 1. can avoid this issue.
For long term, I think we can put the testBackupRestoreSuite into BR integration test, since we merge these two repo together. @bb7133